src/test/resources/examples/openFeignClient/api.yaml

```diff
@@ -169,6 +171,19 @@ components:
       schema:
         type: boolean
       required: false
+    QueryParam4:
```

Comment:
Let's rename these to better indicate that they are header parameters to avoid confusion with the query string parameters

Reply:
Ok, fixed